tests/unit/test_footprinter.cc

```diff
@@ -105,9 +105,8 @@ struct TestDerived2 : TestBase {
   template <
     typename SerializerT,
     typename = std::enable_if_t<
-      std::is_same<
-        SerializerT,
-        checkpoint::Footprinter
+      checkpoint::is_footprinter<
```

Comment:
Can we also get `checkpoint::is_footprinter_v` helper?

Reply:
Done - I missed a spot to swap to is_footprinter_v in the footprinter unit test, but that's handled by #370 now